Getting_Started_with_eva_sub_cli.md

```diff
@@ -0,0 +1,127 @@
+# How to generate inputs for the eva-sub-cli 
+
+The eva-sub-cli tool requires the following inputs:
+
+- One or several valid VCF files
+- Completed metadata spreadsheet
+- Reference genome in fasta format
+
+The VCF file must adhere to official VCF specifications, and the metadata spreadsheet provides contextual information about the dataset. In the following sections, we will examine each of these inputs in detail.
+
+# VCF File
+
+A VCF (Variant Call Format) file is a type of file used in bioinformatics to store information about genetic variants. It includes data about the differences (or variants) between a sample's DNA and a reference genome. Typically, generating a VCF file involves several steps: preparing your sample, sequencing the DNA, aligning it to a reference genome, identifying variants, and finally, formatting this information into a VCF file. The overall goal is to systematically capture and record genetic differences in a standardised format. A VCF file consists of two main parts: the header and the body.
+Header: The header contains metadata about the file, such as the format version, reference genome information, and descriptions of the data fields. Each line in the header starts with a double ##, except for the last header line which starts with a single #.
+
+File format version
+
+```
+##fileformat=VCFv4.2
+##INFO=<ID=DP,Number=1,Type=Integer,Description="Total Depth">
+##FILTER=<ID=PASS,Description="All filters passed">
+##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
+
+```
+
+Body: The body of the VCF file contains the actual variant data, with each row representing a single variant. The columns in the body are : CHROM, POS, ID, REF, ALT, QUAL, FILTER, INFO, FORMAT, Sample Columns
+
+```
+#CHROM  POS  ID  REF  ALT  QUAL  FILTER  INFO  FORMAT  [SampleIDs...]
+``` 
+Here's a small example to illustrate the structure of a VCF file: Example VCF file archived at EVA to be inserted 
```

Comment:
**Example VCF file archived at EVA to be inserted**  is this a TODO ?

Reply:
@nitin-ebi yes still searching for a good example vcf file that we have archived at EVA